src/Controller/CandidateController.php

```diff
@@ -62,15 +64,20 @@ public function edit(Request $request, Candidate $candidate, CandidateRepository
     public function applyToJob(
         Request $request,
         Offer $offer,
-        CandidateRepository $candidateRepository
+        CandidateRepository $candidateRepository,
+        ApplicationRepository $applicationRepo
     ): Response {
         /** @var User $user */
         $user = $this->getUser();
         $candidate = $user->getCandidate();
         $submittedToken = $request->request->get('token');
 
         if ($this->isCsrfTokenValid('apply-offer', $submittedToken)) {
-            $candidate->addOffer($offer);
+            $application = new Application();
+            $application->setApplicationStatus('in-progress');
```

Comment:
ça correspond à quoi le 'in-progress' ? il faudrait à minima une constante et pas noter en dur la string

Reply:
done